src/Int.mo

```diff
@@ -26,6 +26,7 @@ module {
   public func abs(x : Int) : Nat {
     Prim.abs(x)
   };
+  public func fromNat(n : Nat) : Int = n;
```

Comment:
Needs doc .
Maybe we could add a deprecation warning to say its not really necessary (due to subtyping)